src/ZipkinBundle/Middleware.php

```diff
@@ -46,10 +48,11 @@ final class Middleware
     private $spanCustomizers;
 
     /**
-     * @var bool
+     * @var Tracing $tracing
+     * @var LoggerInterface $logger
+     * @var array $tags
+     * @var SpanCustomizer[]|array $spanCustomizers @deprecated
```

Comment:
"Span customizer" was a partial understanding over the existing functionality in brave, we should align to the `HttpTracing` feature. Accepting many `SpanCustomizer`s is also not ideal,  not sure how many users use more than one (if they do) and probably none should. I guess main use case is to be able to obtain the route (as described in https://github.com/jcchavezs/zipkin-instrumentation-symfony#span-customizers).

I'd like to stop supporting this variadic argument and only accept one, that won't break the signature for users (this is PHP) but will definitively be a breaking change if anyone is passing more than one.

Any thoughts @dimonchuk, @nirnanaaa, @iamtankist, @duxet or @Landerstraeten